app/views/internal/path_redirects/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= link_to "Back to All Path Redirects", internal_path_redirects_path, class: "btn btn-primary float-right" %>
+
+<h2>Path Redirect - ID:  <%= @path_redirect.id %></h2>
+<% badge_type =
+     if @path_redirect.version <= 3
+       "success"
+     elsif @path_redirect.version <= 5
+       "warning"
+     else
+       "danger"
+     end %>
+<h4 class="d-inline-block mb-5">
+  Version:
+  <span class="align-top badge badge-<%= badge_type %>"><%= @path_redirect.version %></span>
+</h4>
+
+<%= form_with model: [:internal, @path_redirect], method: :patch, local: true do |form| %>
+  <div class="form-group">
+    <%= form.label :old_path %>
+    <%= form.text_field :old_path, class: "form-control", disabled: true %>
+  </div>
+  <div class="form-group">
+    <%= form.label :new_path %>
+    <%= form.text_field :new_path, class: "form-control" %>
+  </div>
+  <%= form.submit "Update Path Redirect", class: "btn btn-primary" %>
+<% end %>
+
+<hr class="mt-5">
+
+<div class="d-flex justify-content-start">
+  <%= link_to "Destroy Path Redirect", url_for(action: :destroy, id: @path_redirect.id), method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-danger ml-2 float-right" %>
```

Comment:
I'm not sure of the best way to show this, so I've attached two screenshots to help clarify (see below), but I don't believe that these two buttons are aligned.
Buttons:
![Screen Shot 2020-06-01 at 3 56 49 PM](https://user-images.githubusercontent.com/32834804/83459007-c5a42880-a420-11ea-8a03-4892be3b2200.png)

Better view of them not looking aligned:
![Screen Shot 2020-06-01 at 3 57 30 PM](https://user-images.githubusercontent.com/32834804/83459016-c937af80-a420-11ea-8fc0-e51f7677db4a.png)

Reply:
That's somewhat intentional. The `destroy` button is supposed to be in a pseudo "footer". It would look better if we had more buttons and actions in the footer. Since this is Internal, I didn't bother with it.

Reply:
Ahh, I see! Thanks for the explanation. 👍 👍